content/en/building/reference/app-settings/transitions.md

```diff
@@ -87,7 +86,7 @@ Understanding the different types of reports used in the configuration:
 
 `0`, `1` : `new_reports` : `counted_reports` that came in since the previous alert was sent. They haven't been messaged about yet.
 
-#### Configuration
+##### Configuration
```

Comment:
Thanks for pointing this out @jkuester . I was trying to avoid the headings appearing on the right-hand nav since they were too many and making it too long.
<img width="1291" alt="image" src="https://github.com/user-attachments/assets/8bec2496-d9c3-402d-bd90-308dd05b07bb">

Reply:
What do you think?

Reply:
:+1: I could not find any other way that Docsy lets you configure which headings show up in the sidebar.  I am totally fine leaving these as `h5` since we have an actual visual reason for doing it.